charts/velero/values.yaml

```diff
@@ -557,6 +557,10 @@ nodeAgent:
   # labels to set for the node-agent daemonset. Optional.
   labels: {}
 
+  # Additional pod labels for the node-agent daemonset. Optional
+  # ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
+  podLabels: {}
```

Comment:
Isn't there is the label for the node-agent?
https://github.com/vmware-tanzu/helm-charts/blob/f0f07defa8273b0493806032349539f948da40b4/charts/velero/values.yaml#L557-L558